src/Sulu/Component/PHPCR/Tests/Unit/PathCleanupTest.php

```diff
@@ -116,7 +112,7 @@ public function testEmojiCleanup(string $a, string $b, string $locale): void
         $this->assertEquals($b, $clean);
     }
 
-    public function emojiCleanupProvider(): \Generator
+    public static function emojiCleanupProvider(): \Generator
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking change

Reply:
This function doesn't seem to exist in the 2.5 branch.